### ISSUE TITLE
Autocomplete doesn't work on the scratch org

### ISSUE THREAD

**Issue by alexandrgoryachko** [ChuckJonas/vscode-apex-autocomplete#31]
I created scratch org with using sfdx tools. The structure of the files on sfdx project differs from the usual project. I added org creditionals to my workspace settings to get completions on SObject & SOQL Queries. And now I can't get any field from sObject in a SOQL query. Could you find the problem between the scratch org and autocomplete extension.

**Comment by ChuckJonas** [comment]
I don't see any reason why it wouldn't work, as long as you [generated a password](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_passwd.htm) for it and set up the workspace for the extension properly.

I'd try maybe just reloading vscode...

**Comment by alexandrgoryachko** [comment]
@ChuckJonas I reloaded vscode, it doesn't help. The response is empty for scratch org in contrast to usual org:
![joxi](https://user-images.githubusercontent.com/17109581/39981866-170fc838-575b-11e8-94e5-2883b379375a.jpg)

**Comment by ChuckJonas** [comment]
you've already pushed `TG_Timesheet__c` to the scratch org, correct?  Do other types of SObject auto-complete work.  EG:

```
Account acc = new Account();
acc.
```

**Comment by alexandrgoryachko** [comment]
@ChuckJonas Yes, TG_Timesheet__c already pushed the custom object. With standard objects(Account) the problem remains
